test/blame/b-key-navigation-test

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+. libtest.sh
+. libgit.sh
+
+export COLUMNS=200
+
+# This test verifies the fix for issue #1369
+# The test checks that pressing 'b' in blame view correctly navigates
+# to the corresponding line in the target commit.
+
+in_work_dir create_repo_from_tgz "$base_dir/files/scala-js-benchmarks.tgz"
+
+tigrc <<EOF
+set line-graphics = ascii
+EOF
+
+steps '
+	# Press b to go to blame for that line/commit
+	:view-blame
+	:save-display recursive-blame.screen
+'
+
+test_tig blame +27 project/Build.scala
```

Comment:
These aren't really problems, but I copied this test to make a test for the `,` key bug in #1372, and we could coordinate the two tests.

- If you use line 36 here, it works as an example for both `,` and `b` keys
- You could rename the file to something like `navigation-b-key-test`, so that the two tests get placed next to each other. (This seems slightly simpler than combining them into one test).